src/app/student-dash/student-dash.component.html

```diff
@@ -19,8 +23,7 @@
       </mat-form-field>
     </div>
     <div *ngIf="service.isAdmin" fxFlexAlign="baseline">
-      <button mat-raised-button color="primary" 
-          id="create-exam-button" (click)="createExam()">Create Exam!</button>
+      <button mat-raised-button color="primary" id="create-exam-button" (click)="createExam()">Create Exam!</button>
```

Comment:
Change not required. Let us not change the formatting unless very essential, please.